sdk/php/src/Specifications.php

```diff
@@ -28,6 +29,30 @@ class Specifications
      */
     const GROUP_NAME_DEFAULT = 'default';
 
+    /**
+     * Vehicle types models.
+     *
+     * @var string
+     */
+    const VEHICLE_TYPE_AGRICULTURAL = 'agricultural';
```

Comment:
@DenSul Move constants to VehicleModelType class

Reply:
done